api/authentication/service.md

```diff
@@ -0,0 +1,148 @@
+# AuthenticationService
+
+[![npm version](https://img.shields.io/npm/v/@feathersjs/authentication.png?style=flat-square)](https://www.npmjs.com/package/@feathersjs/authentication)
+[![Changelog](https://img.shields.io/badge/changelog-.md-blue.png?style=flat-square)](https://github.com/feathersjs/feathers/blob/master/packages/authentication/CHANGELOG.md)
+
+```
+$ npm install @feathersjs/authentication --save
+```
+
+The `AuthenticationService` is a [Feathers service](../services.md) that allows to register different authentication strategies and manage [JSON web tokens (JWT)](https://jwt.io/).
+
+```js
+const { AuthenticationService, JwtStrategy } = require('@feathersjs/authentication');
+
+module.exports = app => {
+  const authService = new AuthenticationService(app);
+
+  service.register('jwt', new JwtStrategy());
+
+  app.use('/authentication', authService);
+}
+```
+
+## Configuration
+
+## app.get('defaultAuthentication')
+
+After registering an authentication service, it will set the `defaultAuthentication` property on the application to its configuration name (see `configKey` below) if it does not exist. `app.get('defaultAuthentication')` will be used by other parts of Feathers authentication to access the authentication service if it is not otherwise specified.
+
+## constructor(app)
+
+`new AuthenticationService(app, configKey = 'authentication')` initializes a new authentication service with the [Feathers application]() instance and a `configKey` which is the name of the configuration property to use via [app.get()](../application.md#get-name) (default: `app.get('authentication')`). Upon initialization it will also update the configuration with the [default settings](#configuration).
+
+## configuration
+
+`service.configuration` returns a copy of current value of `app.get(configKey)` (default: `app.get('authentication')`). It is usually initialized through an `authentication` property in a [configuration file](../configuration.md) (default: `config/default.json`) which must at least contain a `secret` property. The default configuration is:
+
+```js
+{
+  entity: 'user', // The name of the entity. Can be `null` when not used
+  service: 'users', // The service to retrieve the entity from
+  jwtOptions: {
```

Comment:
This was changed to accessTokenOptions, or not?

Reply:
I think it should still be `jwtOptions` since it is specifically for creating a JWT (Auth0 and most other libraries are calling it the same, too). I just thought I'd rename the methods to `*accessToken` because it doesn't have to be a JWT.